src/helpers/messages.js

```diff
@@ -70,11 +70,16 @@ function getHeader(scope) {
  * @returns {String} Returns the message text
  */
 function getActionMessage(scope, submitButtonLabel = __('OK')) {
+    var msg;
     switch (scope) {
         case 'section':
         case 'testSection':
         case 'part':
-            return `${__('Click "%s" to continue', submitButtonLabel)}.`;
+            msg = __('Click "%s" to continue', submitButtonLabel);
+            if (msg.length > 1) {
```

Comment:
This condition is always true, because you define `msg` one line before and it has a value.

Reply:
Not always, if argument translated to empty string

Reply:
You are right. My bad.